tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -31,6 +31,21 @@ class MatmulATenEvaluationTest : public NVFuserTest {
       optimization_guard_;
 };
 
+using MatmulNodeParamType =
+    std::tuple<std::vector<int64_t>, std::vector<int64_t>>;
```

Comment:
Nit:

```suggestion
using Sizes = std::vector<int64_t>;
using MatmulNodeParamType = std::tuple<Sizes, Sizes>;
```
to save you some typing.